www/Component.js

```diff
@@ -19,6 +19,9 @@ sap.ui.define([
 		init: function() {
 			// call the base component's init function
 			UIComponent.prototype.init.apply(this, arguments);
+			
+			var newModel = new sap.ui.model.json.JSONModel("./model/Operators.json");
```

Comment:
Model (and objects in general) variables should start with o, in this case, it should be oNewModel. Still, it needs a more descriptive name, like oOperatorsModel.